### PR TITLE
Move paths: move paths sdk method with commented out textile call

### PR DESCRIPTION
**Pull request by jsonsivar**
## Description
This implements the move paths on the SDK.  It will take an array of source paths and an array of destination paths and line them up by index and perform the move. It will then produce an event in the returned event emitter for each move operation and then a finally summary with the count of move operations in the end.

Fixes # (issue)
https://app.clubhouse.io/terminalsystems/story/21555/sdk-move

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
NOTE: pending Textile release for true integration test.

- [ ] Unit Test
- [x] Integration Test

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/storage/src/userStorage.ts [hunk 1]
```diff
@@ -911,6 +914,73 @@ export class UserStorage {
...
+  }
+
+  private async moveMultiplePaths(
+    bucketName:string,
+    sourcePaths: string[],
+    destPaths: string[],
+    client: Buckets,
+    bucket: BucketMetadataWithThreads,
+    emitter: ee.Emitter,
+  ) {
```
Comment (perfectmak) [inline packages/storage/src/userStorage.ts:953]:
```suggestion
  ): Promise<MovePathsResultSummary> {
```

**Review by perfectmak — APPROVED** [review]
LGTM

**Review by angarita-dev — APPROVED** [review]
Looks good to me

**Comment by jsonsivar** [comment]
Hey @perfectmak - can you kindly review the latest commit on this. There were some more changes needed after last review.  It's working against dev hub now so we can do a quick review and then I can release.

**Review by perfectmak — APPROVED** [review]
LGTM! Seems CI is failing.

**Pull request merged**